hack/typos.toml

```diff
@@ -1,5 +1,9 @@
 [files]
-extend-exclude = ["go.mod", "go.sum"]
+extend-exclude = ["go.mod", "go.sum", "config/crds/"]
+
+[default]
+extend-ignore-re = ["shortName=[a-z]+"]
```

Comment:
Should we ignore `^//\+.*$` instead?

Reply:
I don't think so. There are no gibberish words behind `//+` in general; it's just a command, and you still want to typo-check commands in general.